tmk_core/common/action.c

```diff
@@ -43,10 +43,14 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 int tp_buttons;
 
-#if defined(RETRO_TAPPING) || defined(RETRO_TAPPING_PER_KEY)
+#if defined(RETRO_TAPPING) || defined(RETRO_TAPPING_PER_KEY) || (defined(RETRO_SHIFT) && !defined(NO_ACTION_TAPPING))
```

Comment:
I don't believe the `NO_ACTION_TAPPING` checks here or just below are necessary.